server/src/index.ts

```diff
@@ -35,7 +35,34 @@ router.use('/customers', customersRouter);
 router.use('/listings', listingsRouter);
 router.use('/merchants', merchantsRouter);
 
-app.use('/', router);
+app.use(router);
+
+// handles server error
```

Comment:
Prefer writing comments with complete sentence, i.e. capital first letter and full stop at the end.

Same as the rest.

Reply:
okay!